data/transactions/logic/README.md

```diff
@@ -615,13 +615,13 @@ Account fields used in the `acct_params_get` opcode.
 | `app_params_get f` | X is field F from app A. Y is 1 if A exists, else 0 |
 | `acct_params_get f` | X is field F from account A. Y is 1 if A owns positive algos, else 0 |
 | `log` | write A to log state of the current application |
-| `box_create` | make a box |
-| `box_extract` | read from a box |
-| `box_replace` | write to a box |
-| `box_del` | delete a box |
-| `box_len` | length of a box |
-| `box_get` | full contents of a box |
```

Comment:
should we mention the 4096 limit for `box_extract` as well?

Reply:
I don't know. I was reluctant to mention it in `box_get`. It's really a result of the general rule that no value can be more than 4096. So should we mention this in all opcodes that could otherwise return more than 4096? Or in none? Or in the ones where you might most likely stumble?

Reply:
I vote for consistency and the simplest way to achieve that is not to mention it in any opcode.

Reply:
I agree. I think what I'm really trying to do is draw attention to the fact that `box_get/put` are conveniences, and that `create/replace/extract` exist for bigger jobs. So perhaps I'll say exactly that in the "extra docs" section.